collection/sorted-array-set.md

```diff
@@ -40,3 +40,14 @@ than 100 values.
 
 `SortedArraySet` instances fly the `isSorted` and `isSet` flags.
 
+> ⚠️ **Callback Caveat**:
+> The equality of your `compare` callback must match the equality of your `equals` callback,
+> or you may end up with duplicates in your set.
+> #### Broken example:    
+> `equals`: `(foo, bar) => foo.a === foo.a`  
+> `compare`: `(foo, bar) => foo.b - foo.b`
+> #### Working alternative:
+>`equals`: `(foo, bar) => foo.a === foo.a`  
+>`compare`: `(foo, bar) => foo.a === foo.b ? 0 : foo.b - foo.b` 
```

Comment:
There are only `foo` variables, there should be `bar` on the right sides I guess.

Reply:
WHEW thanks for the catch, that's embarrassing.